packages/wrangler/bin/wrangler.js

```diff
@@ -2,7 +2,7 @@
 const { spawn } = require("child_process");
 const path = require("path");
 
-const MIN_NODE_VERSION = "18.0.0";
+const MIN_NODE_VERSION = "20.0.0";
```

Comment:
I am not sure we should hard fail on node 18 quite yet. How about a big warning for the next few months?

Reply:
I've made it a warning in [612e8f8](https://github.com/cloudflare/workers-sdk/pull/9112/commits/612e8f887af648c2b58ad26ca931b85c2c87d4dd). The warning is in the Wrangler banner (rather than `wrangler.js`) so that it doesn't get shown on commands that need clean JSON output.